### ISSUE TITLE
How to recursively display files/dir ?

### ISSUE THREAD

**Issue by k0pernicus** [cobalt-org/cobalt.rs#140]
I try to use `posts` as a planning directory.
So, each subdirectory in `posts` is a date, which contains some files and each of these files is a talk and some informations about that one.

So, I have this configurations:

```
post/
    |-> 22-09-2016/
        |-> talk1.md
        ...
    |-> 23-09-2016/
        |-> talk1.md
        ...
```

I think that the `index.liquid` seems to be like that:

```
{% for s_date in posts %}
    // Get the name of the subdirectory to get the date of following talks
    {% for post in s_date %}
        {{ post.content }}
    {% endfor %}
{% endfor %}
```

But I don't think that `s_date` is recognized as a subdirectory from `posts`...
Is there a simple solution for my problem please?

Thanks a lot!

**Comment by uwearzt** [comment]
see pull request https://github.com/cobalt-org/cobalt.rs/pull/134

**Comment by uwearzt** [comment]
can be closed, done with #185
